### ISSUE TITLE
Calculating Final combined Validation Accuracy

### ISSUE THREAD

**Issue by Aravinda89** [yjxiong/caffe#183]
I'm new to deep learning. I use UCF sports dataset (it has 150 sports videos). I wanted to finetune Deep Two-Stream ConvNets for UCF sports dataset. So first I extracted rgb frames & optical flow frames. I used rgb frames & already fine-tuned spatial net, used optical flow frames and fined tuned temporal net. I tested both spatial & temporal net on test set. Now I have two separate validation accuracies **82% for spatial net**, **88% for temporal net**. My question is how can I calculate final "**Combined validation accuracy**"?
Ex: In paper it says... using UCF101 validation split1 there are two validation accuracies: **spatial: 79.8%**, **temporal:85.7%**, "**combined:90.9%**" ... How did they calculate this **combined accuracy**? Please help. Thank **you.**

**Comment by yjxiong** [comment]
We fuse the two modalities by a weighted sum of the classifier scores from each modality, a.k.a. voting.

Please refer to the TSN codebase for if need to know how to implement this.

https://github.com/yjxiong/temporal-segment-networks/blob/master/tools/eval_scores.py#L38

**Comment by Aravinda89** [comment]
As I understood..correct me if I'm wrong, first have to generate 2 classifier scores files for each temporal net and spatial net. Then use "tool/eval_scores.py" to get fusion result. Question is how to generate these classifier scores files? Should I modify "tool/eval_net.py"? or Is there another way? 
Anyone faced same problem?

Another question, This script 
https://github.com/yjxiong/caffe/blob/action_recog/action_python/demo.py
What is this script for? Is it predicting label for single video?

**Comment by yjxiong** [comment]
1. Yes. The `eval_net.py`  in TSN will generate the score files. See 
https://github.com/yjxiong/temporal-segment-networks/wiki/Working-on-custom-datasets.
on how to use your own dataset.

2. This script is contributed by the users. It gives the predicted label for a single video.
